src/Hangfire.Core/BackgroundJobServer.cs

```diff
@@ -163,5 +174,47 @@ public void Start()
         public void Stop()
         {
         }
+
+        public void OnNext(int i)
```

Comment:
The interface didn't allow us to implement it without a type, "OnNext()" would had been preffered but did not work :/